jetstack-index.yaml

```diff
@@ -0,0 +1,1098 @@
+apiVersion: v1
```

Comment:
This file should be generated/pulled on demand rather than committed to the repo.

Reply:
I agree. However, this file and `repositories.yaml` are supposed to be generated when we do a `helm repo add`. But since every helm command is run by a separate container, we need to conserve the state after `helm repo add`. Is it possible to have a single container execute both `helm repo add` and `helm install cert-manager`?